marathon/resource_marathon_app.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strconv"
 	"time"
 
+	cosmos "github.com/PTC-Global/dcos-sdk-go/cosmos/api/lib"
```

Comment:
shouldn't we first create a PR for squeakysimple/dcos-sdk-go and use that here?

Reply:
I created that pr and It's actually merged. But I think we may need to wait for the licensing issue, before changing this to squeakysimple/dcos-sdk-go

Reply:
we can keep this temporarily until we get our changes in `squeakysimple/dcos-sdk-go`

Reply:
We created a pr for adding MIT license to the original repo. I'm updating the pr